cfme/tests/infrastructure/test_quota_tagging.py

```diff
@@ -17,8 +18,11 @@ def vm_name():
 
 
 @pytest.fixture
-def template_name(provisioning):
-    return provisioning["template"]
+def template_name(provider):
+    if provider.type == 'rhevm':
+        return provider.data.templates.get('full_template')['name']
+    elif provider.type == 'virtualcenter':
+        return provider.data.templates.get('big_template')['name']
```

Comment:
**REQUIRED**
`provider.one_of(RHEVMProvider)` and `provider.one_of(VMwareProvider)` instead of `provider.type == ...`

Reply:
Changed.